src/reducers/results.js

```diff
@@ -18,6 +18,18 @@ const tryGroupedResultCount = (data) => {
   return 0;
 };
 
+const tryFacetRanges = (data) => {
+  var newDict = {};
+  if (data.facet_counts.facet_ranges) {
+    let facet_ranges = data.facet_counts.facet_ranges;
+    for (let key in facet_ranges) {
+      newDict[key] = facet_ranges[key]["counts"];
+    }
+    return newDict;
+  }
+  return [];
+};
+
```

Comment:
Unfortunately, the date facets show up in another key in the results, so we have to parse them out differently to get them presented to the UI components in the same way.